Makefile

```diff
@@ -12,6 +12,22 @@ test:
 	.venv/bin/py.test tests
 
 clean:
-	rm -Rf .venv 
+	rm -Rf .venv
 
-.PHONY: install test
+# Works with bash and linux
+load-tests:
+	cp conf/*.ini .
+	curl -o reingart.zip https://www.sistemasagiles.com.ar/soft/pyafipws/reingart.zip
+	python -m zipfile -e reingart.zip .
+
+sign-tra:
```

Comment:
This target obtains an access ticket (create the access ticket request, signs it, and calls the webservice to get the final access ticket)
Maybe it would be better to use a more descriptive name like `try-auth:`

Please add comments or echo to document these steps inline.